bucket/mremoteng-portable.json

```diff
@@ -0,0 +1,39 @@
+{
+    "homepage": "https://mremoteng.org/",
+    "description": "An open source, tabbed, multi-protocol, remote connections manager.",
+    "version": "1.76.20.24669",
+    "hash": "97b0e94bbdc409f3eb09b1017b3b58ccc0eae145a98a3bc6ea651ef1d5bc2d8e",
+    "license": "GPL-2.0-only",
+    "url": "https://github.com/mRemoteNG/mRemoteNG/releases/download/v1.76.20/mRemoteNG-Portable-1.76.20.24669.zip",
+    "bin": "mRemoteNG.exe",
+    "extract_dir": "",
```

Comment:
```suggestion
```

Reply:
For some reason it failed when not set... may be I did something wrong. When I added the key with "", then it worked ok.